packages/core/core/src/AssetGraphBuilder.js

```diff
@@ -123,25 +105,10 @@ export default class AssetGraphBuilder extends EventEmitter {
       onNodeRemoved: node => this.handleNodeRemovedFromAssetGraph(node),
     });
 
-    let assetGraph = this.assetGraph;
     this.requestTracker = new RequestTracker({
       graph: this.requestGraph,
-    });
-    let tracker = this.requestTracker;
-    this.entryRequestRunner = new EntryRequestRunner({
-      tracker,
-      options,
-      assetGraph,
-    });
-    this.targetRequestRunner = new TargetRequestRunner({
-      tracker,
-      options,
-      assetGraph,
-    });
-    this.configRequestRunner = new ParcelConfigRequestRunner({
-      tracker,
-      options,
-      workerFarm,
+      farm: workerFarm,
+      options: this.options,
```

Comment:
Eventually I'm thinking there could be an 'options_request' with the specific options the request is interested in and then we don't have to invalidate the entire graph when any of the options change.